packages/backend/src/functions/ceremony.ts

```diff
@@ -15,10 +15,10 @@ export const startCeremony = functions.pubsub.schedule(`every 30 minutes`).onRun
     // Get ceremonies in `scheduled` state.
     const scheduledCeremoniesQuerySnap = await queryCeremoniesByStateAndDate(CeremonyState.SCHEDULED, "startDate", "<=")
 
-    if (scheduledCeremoniesQuerySnap.empty) logMsg(GENERIC_LOGS.GENLOG_NO_CEREMONIES_READY_TO_BE_OPENED, MsgType.INFO)
+    if (scheduledCeremoniesQuerySnap.empty) printLog(`No ceremonies ready to be opened`, LogLevel.INFO)
```

Comment:
Maybe we can remove this log and just log when we start a ceremony?

Reply:
Totally missed the refactoring of these two functions. Let me add in the next commit! Thanks for the heads up!